src/components/InteractiveQuestionView.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Flex, Group, Modal } from "@mantine/core"
+import { InteractiveQuestion } from "@metabase/embedding-sdk-react"
+
+import { ThemedButton } from "./ThemedButton"
+import { CustomIcon } from "./CustomIcon"
+import { useReducer } from "react"
+import { useDisclosure } from "@mantine/hooks"
+
+// TODO: add the "breakout" view once we've separated the breakout from summarization in the SDK
+type QuestionView = "viz" | "filter" | "summary" | "editor"
+
+interface Props {
+  isSaveEnabled?: boolean
+}
+
+export const InteractiveQuestionView = (props: Props) => {
+  const { isSaveEnabled = false } = props
```

Comment:
We only enable the "Save" button in the "new question from template" page for now.

Reply:
why do you destructure props on some components but not all? I can understand in case you need to access the whole `props` for spreading for example. But I don't see the reason to do that here. I found using `props` needs a bit more work to figure out where the all the props are used.

Reply:
Whoops, bad habit. I usually do not destructure the `props` in place. Fixed in https://github.com/metabase/shoppy/pull/80/commits/9f82f07cec66a35a295b5c6d7cece69caeafc689